user/for-beginners.md

```diff
@@ -1,57 +1,53 @@
 
-4. Take a look at `.travis.yml`, the file which tells Travis CI what to do:
+You can also customize your build into various stages, in which later stages are dependent on the result of prior stages. See our [Build Stages](../build-stages) document for details. 
 
-   ```yaml
-   language: php
-   php:
-   - 5.5
-   - 5.4
-   - hhvm
-   script: phpunit Test.php
-   ```
+### Jobs vs. builds
 
-   This file tells Travis CI that this project is written in PHP, and to test `Test.php` with phpunit against PHP versions 5.5, 5.4 and HHVM.
+The words "job" and "build" can be somewhat ambiguous in the context of CI. In the Travis CI docs and materials, they refer to the following: 
+ * *job* - a single automated process, where Travis CI starts an automated worker which clones your projects, compiles your code, runs tests, and executes any other steps, such as deployments. The status of the job depends on the success of the steps it executes. 
+ * *build* - a group of automated processes, where multiple jobs might happen in parallel. For example, a build might have two jobs, which test a project with two different versions of a language. The status of a build depends on the status of the jobs it contains. 
 
-5. Edit the empty `NewUser.txt` file by adding your name to the empty file. Add the file to git, commit and push, to trigger a Travis CI build:
+### travis-ci.org vs travis-ci.com
 
-   ```bash
-   $ git add -A
-   $ git commit -m 'Testing Travis CI'
-   $ git push
-   ```
+Travis CI originally was developed for open-source projects before being expanded to support closed-source projects at a later date. As a result, open-source projects are hosted on our [travis-ci.org](https://travis-ci.org/) platform while closed source projects are hosted on [travis-ci.com](https://travis-ci.com/). Travis CI for closed-source projects is also know as [Travis Pro](../travis-pro/).
 
-   > Note: Travis only runs a build on the commits you push after adding the repository to Travis.
 
-   Wait for Travis CI to run a build on your fork of the `travis-broken-example` repository, check the [build status](https://travis-ci.org/repositories) and notice that the build fails. (Travis CI sends you an email when this happens)
+### Breaking the Build
+--> TBD
 
-6. Fix the code by making sure that `2=1+1` in `Test.php`, commit and push to GitHub. This time, the build does not fail.
+### Infrastructure and environment notes
 
-   ```bash
-   $ git add -A
-   $ git commit -m 'Testing Travis CI: fixing the build'
-   $ git push
-   ```
+Travis CI supports a few different environment, so you have different options to use. Each job runs in exactly one of the following environments:
```

Comment:
Travis CI runs your build in *one* of the following environments, each with some advantages and disadvantages: